integration-scripts/package.json

```diff
@@ -48,8 +47,7 @@
     "eslint": "6.3.0",
     "typechain": "1.0.3",
     "typechain-target-ethers": "^1.0.1",
-    "typescript": "^3.6.3",
-    "@0x/sol-compiler": "^3.1.15"
```

Comment:
Aren't we migrating towards the @0x stuff?

Reply:
Yeah, i was removing these entries because they're duplicates. already under the `dependencies` key